cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -322,6 +324,10 @@ spec:
           args:
           {{- toYaml .Values.kubecostModel.extraArgs | nindent 12 }}
         {{- end }}
+          securityContext:
+          {{- if .Values.kubecostModel.securityContext -}}
```

Comment:
Shouldn't the if statement go above the "securityContext" section to avoid an unnecessary empty yaml block?